import-service/handlers/importFileParser.js

```diff
@@ -0,0 +1,34 @@
+const csv = require('csv-parser')
+
+const AWS = require('aws-sdk');
+const s3 = new AWS.S3({region: 'eu-central-1'});
+const BUCKET = 'nodejs-products-bucket';
+
+export const importFileParser = (event) => {
+    console.log('Lambda importFileParser invocation with event: ', event);
+    event.Records.forEach(record => {
+        console.log("in records parsing file with key=" + record.s3.object.key);
```

Comment:
single and double quotes side by side
better:
`in records parsing file with key=${record.s3.object.key}`

Reply:
I'm a java developer) We have double quotes , i'll try to avoid quotes mess in future. Thank you!